config/env/test.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
 	db: {
-		uri: 'mongodb://localhost/mean-test',
+    		uri: process.env.MONGOHQ_URL || process.env.MONGOLAB_URI || 'mongodb://' + (process.env.DB_1_PORT_27017_TCP_ADDR || 'localhost') + '/mean-test',
```

Comment:
There are either tabs or spaces but unnecessary extra spaces. GitHub shows them, so I would just delete the line and press enter and it should be better.